app/services/pricers/pricer.rb

```diff
@@ -0,0 +1,11 @@
+module Pricers
+  class Pricer
+    # @param price [Result<Float>]
+    # @param params [Hash]
+    # @param now [Time]
+    # @return [Result<Float>]
+    def compute(price, params, now)
+      raise "Not implemented!"
```

Comment:
Did you consider raising `NotImplementedError`?

Reply:
would be better indeed, didn't focused much on this ;)